Dockerfile.gitlab

```diff
@@ -61,14 +61,16 @@ RUN mkdir /root/.android \
 
 # Android SDK
 RUN \
-    wget --quiet --output-document=android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_SDK_TOOLS}.zip && \
-    unzip -d android-sdk-linux android-sdk.zip && \
-    echo y | android-sdk-linux/tools/bin/sdkmanager "platforms;android-${ANDROID_COMPILE_SDK}" >/dev/null && \
-    echo y | android-sdk-linux/tools/bin/sdkmanager "platform-tools" >/dev/null && \
-    echo y | android-sdk-linux/tools/bin/sdkmanager "build-tools;${ANDROID_BUILD_TOOLS}" >/dev/null && \
-    echo y | android-sdk-linux/tools/bin/sdkmanager --install "ndk;${NDK_VERSION}" >/dev/null && \
-    echo y | android-sdk-linux/tools/bin/sdkmanager --install "cmake;${CMAKE_VERSION}" >/dev/null && \
-    yes | android-sdk-linux/tools/bin/sdkmanager --licenses
+    wget --quiet --output-document=android-sdk.zip https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_TOOLS}_latest.zip && \
+    mkdir -p android-sdk-linux/cmdline-tools && \
+    unzip -d android-sdk-linux/cmdline-tools android-sdk.zip && \
+    mv android-sdk-linux/cmdline-tools/cmdline-tools android-sdk-linux/cmdline-tools/latest && \
```

Comment:
why do you have to move it in a `latest` dir ?

Reply:
because this is weird requirement by the `sdkmanager`: it should be either under `cmdline-tools/latest` or `cmdline-tools/<version>` to work, otherwise error :)